spago.dhall

```diff
@@ -0,0 +1,22 @@
+{ name = "spec-reporter-codewars"
+, dependencies =
+  [ "aff"
+  , "bigints"
+  , "console"
+  , "datetime"
+  , "debug"
+  , "effect"
+  , "exceptions"
+  , "maybe"
+  , "prelude"
+  , "profunctor-lenses"
+  , "quickcheck"
+  , "rationals"
+  , "spec"
+  , "spec-discovery"
+  , "spec-quickcheck"
+  , "strings"
+  ]
```

Comment:
:information_source: Some of these dependencies are not necessary. Same with `bit-integers` in package json. I assumed there is a reason for them being in there, so I kept  them.

```
[warn] None of your project files import modules from some projects that are in the direct dependencies of your project.
These dependencies are unused. To fix this warning, remove the following packages from the list of dependencies in your config:
- bigints
- console
- debug
- profunctor-lenses
- rationals
```

Reply:
`package.json` and `psc-package.json` I posted includes dependencies not necessary for the reporter because those are for the project used to test submission.
See https://github.com/codewars/purescript/tree/main/workspace

I'd like to package this reporter properly, so that project can simply depend on it.

Reply:
Can you remove all the unnecessary dependencies?

Reply:
Dropped d1f0c324630d40b4493c105188a9d7cd048bffb1